.github/workflows/conformance.yml

```diff
@@ -35,12 +35,18 @@ on:
         description: Always upload logs regardless of the test result.
         type: boolean
         default: false
+      runner:
+        description: The Github runner to use for the workflow
+        default: ubuntu-latest
+      antrea-image-platform:
+        description: Platform argument to provide when building the Antrea images
+        default: "linux/amd64"
```

Comment:
Can these two be `type: choice`? So we don't have to figure what's other valid options if we want to run the conformance test manually with non-default values.